frontend/src/util/localDocStore.js

```diff
@@ -108,6 +116,7 @@ export const getLocalDocuments = () => {
       owner: userId,
       documentBody: obj.state,
       comments: obj.comments || [],
+      docState: obj.docState ? obj.docState : DOC_STATE.New,
```

Comment:
very much a nit but maybe we want to call it `status` and not `docState` bc that's easy to mix up with `documentBody`